homeassistant/components/devialet/coordinator.py

```diff
@@ -27,7 +30,16 @@ def __init__(self, hass: HomeAssistant, client: DevialetApi) -> None:
             update_interval=SCAN_INTERVAL,
         )
         self.client = client
+        self.entry = entry
```

Comment:
it's already stored in `self.config_entry`, please pass it to the super constructor and overwrite the type

Reply:
Yes, done